trollflow2/launcher.py

```diff
@@ -264,6 +305,23 @@ def get_dask_client(config):
     return client
 
 
+def qlogged_process(msg, prod_list, produced_files, log_queue):
```

Comment:
Should this have a more descriptive name?
```suggestion
def queue_logged_process(msg, prod_list, produced_files, log_queue):
```